### ISSUE TITLE
Add IDs to sections for easy reference

### ISSUE THREAD

**Issue by trezy** [marionettejs/marionettejs.com#409]
One of my coworkers asked my yesterday who actually uses Marionette. I just referred him to the list of the bottom of the website but I would rather have been able to send him a link like [http://marionettejs.com/#logos].

**Comment by peterblazejewicz** [comment]
:+1: 
But on small screens there will be a hidden trap - the navigation will work (they user will be navigated) but this specific section is hidden on mobile screens.
So if section is to be bookmarkable it has to be accessible.
/cc
@jdaudier

**Comment by jdaudier** [comment]
@peterblazejewicz Meaning we have to un-hide it for mobile, correct? Just making sure I am on the same page.

thanks @trezy for the idea!  I'll see you tomorrow at JSConf.

**Comment by peterblazejewicz** [comment]
@jdaudier Yes, just make sure that any header/section with added bookmarkable anchor is visible on all viewports :D

**Comment by jdaudier** [comment]
you got it!

**Comment by trezy** [comment]
@peterblazejewicz I disagree. I've verified on several mobile devices -- Android 4.something, Android 4.somethingElse, iOS8, etc -- that the browser just brings you to the top of the homepage if the section is hidden. No unexpected consequences. I'll dig in and add these this weekend since I got the repo to build this time. :-)

**Comment by peterblazejewicz** [comment]
@trezy that's the point - user arepointed nowhere:
http://bradfrost.com/blog/post/five-responsive-web-design-pitfalls-to-avoid/
> don’t penalise users for the device they happen to be browsing with.

**Comment by trezy** [comment]
@peterblazejewicz That's an issue with us hiding sections. If you're concerned with users being penalized then we should make sure all of the sections are available, i.e. not hidden from mobile. That's a completely separate issue. This is an enhancement to make the page more navigable for those that have the capability.
